### PR TITLE
SC-5361: Running tests

### PR DESCRIPTION
**Pull request by alexanderM91**
### Description

- Ticket/Issue: https://spryker.atlassian.net/browse/SC-5361

### CODE REVIEW

File: docs/02-development-usage/06-running-tests.md [hunk 1]
```diff
@@ -30,16 +30,98 @@ The structure below is just a reference. We encourage you to add subsections, ch
...
+You have two ways of running a testing mode:
+* running `docker/sdk up` with `-t` flag, this command restarts all your containers in the testing environment;
+* running  `docker/sdk testing`, starts a new container where you can run CLI commands in the testing environment.
 
 ### How do I run tests?
 
+The test running process in docker doesn't have any difference from local environment. All you need is to run `codecept run` in CLI container.
+
+You have a few ways:
+* if you run `docker/sdk up` with `-t` flag, you need to go in CLI container(`docker/sdk cli`) and run test;
```
Comment (andriitserkovnyi) [inline docs/02-development-usage/06-running-tests.md:50]: don't I need to run `codecept run` in this case?

File: docs/02-development-usage/06-running-tests.md [hunk 2]
```diff
@@ -30,16 +30,98 @@ The structure below is just a reference. We encourage you to add subsections, ch
...
+* if you start CLI container in testing mode(`docker/sdk testing`), you need is run `codecept run`;
+* you can run `docker/sdk testing codecept run`.
```
Comment (andriitserkovnyi) [inline docs/02-development-usage/06-running-tests.md:52]: In what case do I use this way and what does this command do?

File: docs/02-development-usage/06-running-tests.md [hunk 3]
```diff
@@ -30,16 +30,98 @@ The structure below is just a reference. We encourage you to add subsections, ch
...
+                    args: ["--headless", "--no-sandbox", "--disable-dev-shm-usage"]
+                    binary: "%SPRYKER_TEST_BROWSER_BIN%"
+        \SprykerTest\Shared\Testify\Helper\SuiteFilterHelper:
+            inclusive:
+                - Presentation
+
+params:
+    - tests/default.yml
+    - env
+```
```
Comment (andriitserkovnyi) [inline docs/02-development-usage/06-running-tests.md:126]: What's the successful result of this configuration? What can I with it in the end?

File: docs/02-development-usage/06-running-tests.md (hunk 2 above)
Comment (zyuzka) [reply docs/02-development-usage/06-running-tests.md:51]: 1. docker/sdk testing - running CLI container with testing mode
2. codecept run - running tests

In what case do I use this way?
If you want to run tests in one motion=)

File: docs/02-development-usage/06-running-tests.md (hunk 3 above)
Comment (zyuzka) [reply docs/02-development-usage/06-running-tests.md:109]: This is configuration for web driver

File: docs/02-development-usage/06-running-tests.md [hunk 4]
```diff
@@ -30,16 +30,115 @@ The structure below is just a reference. We encourage you to add subsections, ch
...
-### How do I run tests?
 
+### How do I run tests in the testing mode?
 
-### How do I choose a webdriver?
+To run tests, you need to run Codeception in a CLI container.
+
+There are several ways to do that.
+
+#### Running tests with all containers in testing mode
```
Comment (dmytromykhailov) [inline docs/02-development-usage/06-running-tests.md:45]: Can we explain when all containers in testing mode should be used and when a dedicated container in a testing mode? How a user should pick one solution? What are the benefits of each approach?

File: docs/02-development-usage/06-running-tests.md [hunk 5]
```diff
@@ -30,16 +30,115 @@ The structure below is just a reference. We encourage you to add subsections, ch
 
 ### What is a testing mode?
 
-#### How is the testing mode different from the usual mode?
+Docker SDK allows you to run application in an environment configured for testing.
 
-#### How do I turn on the testing mode?
+In the testing mode, docker/sdk generates an environment file and a set of containers configured for testing. For example, the environment does not have SSL encryption, there is no scheduler container, but the webdriver container is present.
```
Comment (sprymiker) [inline docs/02-development-usage/06-running-tests.md:35]: Please, mention environment variable SPRYKER_TESTING_ENABLED.

Comment (sprymiker) [reply docs/02-development-usage/06-running-tests.md:35]: `environment file` is not true. We have only env var.

Please remove 'the environment does not have SSL encryption'.

`there is no scheduler container` -> background jobs are disabled.

Please, make all features as a bulleting list.

File: docs/02-development-usage/06-running-tests.md [hunk 6]
```diff
@@ -30,16 +30,115 @@ The structure below is just a reference. We encourage you to add subsections, ch
...
-### How do I choose a webdriver?
+To run tests, you need to run Codeception in a CLI container.
+
+There are several ways to do that.
+
+#### Running tests with all containers in testing mode
+
+To run tests with all containers in testing mode:
+
+1. Restart all containers in testing mode:
```
Comment (alexanderM91) [inline docs/02-development-usage/06-running-tests.md:49]: Could you please add more examples of how to run speciifc   tests groups?

**Pull request merged**